src/index.js

```diff
@@ -1,3 +1,51 @@
-import MUIDataTable from './MUIDataTable';
+import MUIDataTable from "./MUIDataTable";
+import MUIDataTableBody from "./MUIDataTableBody";
+import MUIDataTableBodyCell from "./MUIDataTableBodyCell";
+import MUIDataTableBodyRow from "./MUIDataTableBodyRow";
+import MUIDataTableFilter from "./MUIDataTableFilter";
+import MUIDataTableFilterList from "./MUIDataTableFilterList";
+import MUIDataTableHead from "./MUIDataTableHead";
+import MUIDataTableHeadCell from "./MUIDataTableHeadCell";
+import MUIDataTableHeadRow from "./MUIDataTableHeadRow";
+import MUIDataTablePagination from "./MUIDataTablePagination";
+import MUIDataTableResize from "./MUIDataTableResize";
+import MUIDataTableSearch from "./MUIDataTableSearch";
+import MUIDataTableSelectCell from "./MUIDataTableSelectCell";
+import MUIDataTableToolbar from "./MUIDataTableToolbar";
+import MUIDataTableToolbarSelect from "./MUIDataTableToolbarSelect";
+import MUIDataTableViewCol from "./MUIDataTableViewCol";
 
 export default MUIDataTable;
+export {
+  MUIDataTable,
+  MUIDataTableBody,
+  MUIDataTableBodyCell,
+  MUIDataTableBodyRow,
+  MUIDataTableFilter,
+  MUIDataTableFilterList,
+  MUIDataTableHead,
+  MUIDataTableHeadCell,
+  MUIDataTableHeadRow,
+  MUIDataTablePagination,
+  MUIDataTableResize,
+  MUIDataTableSearch,
+  MUIDataTableSelectCell,
+  MUIDataTableToolbar,
+  MUIDataTableToolbarSelect,
+  MUIDataTableViewCol,
```

Comment:
@Legogris looks like the "suggested change" feature is wonky. You'll have to delete the above.